init.sh

```diff
@@ -53,7 +53,13 @@ export EXISTING_PUBLIC_SUBNET1_ID=${46}
 export EXISTING_PUBLIC_SUBNET2_ID=${47}
 export EXISTING_PUBLIC_SUBNET3_ID=${48}
 export PRIVATE_CLUSTER=${49}
-export ENV_TYPE=prod
+
+if [ -z ${50} ] || [ ${50} == "prod" ]; then
+  export ENV_TYPE="prod"
```

Comment:
You can assign the value to the variable directly.
`export ENV_TYPE=${50}`

Reply:
The reason for not assigning directly `export ENV_TYPE=${50}` -  is to maintain a safeguard mechanism so that, just in case if any of the other templates have not been assigned any parameter value (although I did check through the UPI and the use existing OCP), the script wont bomb.
Consider a case wherein there is no value assigned from `CFT`, in that kind of scenario, the `$ENV_TYPE` will have a null value. The `if` condition would have prevented that kind of scenario as by default it will always be assigned some value.